src/crypto/keytypes.rs

```diff
@@ -10,27 +10,27 @@ use std::fmt;
 // 3. Add your key to the list of consts below for signature length
 // 4. add your constant to the macro sig_size.
 pub enum PublicKey {
-  OpenSSLVeryLow(openssl::pkey::PKey<openssl::pkey::Public>),
-  OpenSSLLow(openssl::pkey::PKey<openssl::pkey::Public>),
-  OpenSSLMedium(openssl::pkey::PKey<openssl::pkey::Public>),
-  OpenSSLHigh(openssl::pkey::PKey<openssl::pkey::Public>),
-  Ed25519(ed25519::PublicKey, ed25519::PublicKey), // First key is encryption key, second is verification key
+    OpenSSLVeryLow(openssl::pkey::PKey<openssl::pkey::Public>),
+    OpenSSLLow(openssl::pkey::PKey<openssl::pkey::Public>),
+    OpenSSLMedium(openssl::pkey::PKey<openssl::pkey::Public>),
+    OpenSSLHigh(openssl::pkey::PKey<openssl::pkey::Public>),
+    Ed25519(ed25519::PublicKey, ed25519::PublicKey), // First key is encryption key, second is verification key
 }
 
 pub enum PrivateKey {
-  OpenSSLVeryLow(openssl::pkey::PKey<openssl::pkey::Private>),
-  OpenSSLLow(openssl::pkey::PKey<openssl::pkey::Private>),
-  OpenSSLMedium(openssl::pkey::PKey<openssl::pkey::Private>),
-  OpenSSLHigh(openssl::pkey::PKey<openssl::pkey::Private>),
-  Ed25519(ed25519::SecretKey, ed25519::SecretKey), // First key is encryption key, second is verification
+    OpenSSLVeryLow(openssl::pkey::PKey<openssl::pkey::Private>),
+    OpenSSLLow(openssl::pkey::PKey<openssl::pkey::Private>),
+    OpenSSLMedium(openssl::pkey::PKey<openssl::pkey::Private>),
+    OpenSSLHigh(openssl::pkey::PKey<openssl::pkey::Private>),
+    Ed25519(ed25519::SecretKey, ed25519::SecretKey), // First key is encryption key, second is verification
 }
 
 // The length of the used signatures              pyipv8 | .size() | diff | bits/8*2
-const VERY_LOW_SIGNATURE_LENGTH: usize = 42;  //  42     |   50    |  8   | 42
-const LOW_SIGNATURE_LENGTH:      usize = 60;  //  60     |   66    |  6   | 58
-const MEDIUM_SIGNATURE_LENGTH:   usize = 104; //  104    |   110   |  6   | 102
-const HIGH_SIGNATURE_LENGTH:     usize = 144; //  144    |   153   |  9   | 144
-const ED25519_SIGNATURE_LENGTH:  usize = 64;  //  64     |   64    |  0   | 64
+const VERY_LOW_SIGNATURE_LENGTH: usize = 42; //  42     |   50    |  8   | 42
+const LOW_SIGNATURE_LENGTH: usize = 60; //  60     |   66    |  6   | 58
+const MEDIUM_SIGNATURE_LENGTH: usize = 104; //  104    |   110   |  6   | 102
+const HIGH_SIGNATURE_LENGTH: usize = 144; //  144    |   153   |  9   | 144
+const ED25519_SIGNATURE_LENGTH: usize = 64; //  64     |   64    |  0   | 64
```

Comment:
We want to either let rustfmt ignore this here or place the table in a proper doc comment, I'm in favour of the latter.